zeebe.js

```diff
@@ -0,0 +1,96 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.ZEEBE_CLIENT_ID,
+  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
+  audience: process.env.ZEEBE_AUDIENCE
+};
+
+// An action that assigns a user task.
+async function assignUser([assignee]) {
+    // Every request needs an access token.
+    const accessToken = await getAccessToken(authorizationConfiguration);
+  
+    // These settings come from your .env file.
+    const ZeebeApiUrl = process.env.ZEEBE_BASE_URL;
+    const userTaskKey = process.env.USER_TASK_KEY;
+  
+    // This is the API endpoint to assign a user task.
+    const url = `${ZeebeApiUrl}/user-tasks/${userTaskKey}/assignment`;
+  
+    // Configure the API call.
+    const options = {
+      method: "POST",
+      url,
+      headers: {
+        Accept: "application/json",
+        Authorization: `Bearer ${accessToken}`
+      },
+      data: {
+        // The body contains information about the new assignment.
+        assignee: assignee
+      }
+    };
+  
+    try {
+      // Call the add endpoint.
+      const response = await axios(options);
+  
+      // Process the results from the API call.
+      const assignee = response.data;
+  
+      console.log(
+        `User task assigned to ${assignee.assignee}.`
+      );
+    } catch (error) {
+      // Emit an error from the server.
+      console.error(error.message);
+    }
+  }
+
+async function unassignUser([assignee]) {
```

Comment:
```suggestion
async function unassignTask([assignee]) {
```

Related to my above comment re subject vs object; unrelated to the other comment on this line, about arguments.